src/libcrun/linux.c

```diff
@@ -465,12 +563,15 @@ do_mount (libcrun_container_t *container,
               if (UNLIKELY (ret < 0))
                 return ret;
 
-              ret = mount ("/sys", to, "/sys", MS_BIND | MS_REC | MS_SLAVE, data);
-              if (LIKELY (ret == 0))
-                return 0;
+              if (ret > 0)
+                {
+                  ret = mount ("/sys", to, "/sys", MS_BIND | MS_REC | MS_SLAVE, data);
```

Comment:
Seems it would be better by using
```
ret = mount ("/sys", to, fstype, MS_BIND | MS_REC | MS_SLAVE, data);
```

Reply:
I think the fstype is ignored when `MS_BIND` is specified in the options.

Could you please give more information?  Is it causing any issue?

Reply:
Yeah, the `fstype` is ignored for bind mount. But it took me a while to figure out that the second "/sys" is actually `fstype`:) So feel it would help to understand the code by replacing the second "/sys" with `fstype`.
No functional difference anyway.

Reply:
BTW, any plan to integrate this feature into runC?

Reply:
I've checked the current code and that code chunk seems clearer now:

```
ret = mount ("/sys", real_target, NULL, MS_BIND | MS_REC, NULL);
```

not sure about runC plans, runc and crun are two distinct (and competing) projects.

Reply:
Yeah, much better now:)
For runC and crun, another competitor, youki?

Reply:
yes, youki is another good competitor :-)